apps/ceramic/src/models/page.graphql

```diff
@@ -11,14 +6,15 @@ type PageNode {
 }
 
 type Page @createModel(accountRelation: LIST, description: "Page Object") {
-  type: PageType!
-  title: String! @string(maxLength: 128)
+  type: String! @string(maxLength: 1000000) # using string to allow encryption
+  key: String @string(maxLength: 1024)
+  title: String! @string(maxLength: 1000000)
   data: [PageNode!] @list(maxLength: 100000)
   version: CommitID! @documentVersion
   createdBy: DID! @documentAccount
-  createdAt: DateTime!
+  createdAt: String! @string(maxLength: 1000000)
```

Comment:
Is this an epoch/unix timestamp?

Reply:
ISO 8601, but could store it as whatever. Need to have it as a `String` since we might encrypt it